.ci.yaml

```diff
@@ -3011,14 +3011,14 @@ targets:
       task_name: service_extensions_test
     scheduler: luci
 
-  - name: Mac_android smoke_catalina_start_up
+  - name: Mac_android flutter_gallery_mac__start_up
```

Comment:
Based on https://github.com/flutter/flutter/wiki/Adding-a-new-Test-Shard#steps-to-add-a-new-framework-test-shard, should we be duplicating these with `bringup: true` to let the name soak in?

Reply:
Based on the above doc, shall we update the `framework_config.star` specifically, or simply update `.ci.yaml` with `bringup:true` to wait for the roll?

Reply:
Good point :-)

Can you update the docs, and mention that renaming a target that is postsubmit only doesn't require following the bringup process?

Reply:
Will update in a separate PR.

Reply:
Added a note at the bottom: https://github.com/flutter/flutter/wiki/Adding-a-new-Test-Shard